packages/web/src/js/__tests__/FileUploader.test.ts

```diff
@@ -0,0 +1,88 @@
+import { clearFixture, getFixture } from '../../../tests/helpers/fixture';
+import FileUploader from '../FileUploader';
+
+describe('FileUploader', () => {
+  let fixtureEl: Element;
+  let container: HTMLElement;
+  let input: HTMLInputElement;
+  let instance: FileUploader;
+
+  beforeEach(() => {
+    container = document.createElement('div');
+    input = document.createElement('input');
+    input.type = 'file';
+    container.appendChild(input);
+    instance = new FileUploader(container);
+    fixtureEl = getFixture();
+  });
+
+  afterEach(() => {
+    instance.clearFileQueue();
+    clearFixture();
+  });
+
+  test('should be an instance of FileUploader', () => {
```

Comment:
We are using `it` instead of `test`. Please fix it also on other locations.

```suggestion
  it('should be an instance of FileUploader', () => {
```

Reply:
fixed